### ISSUE TITLE
Crash NSGenericException : *** Collection <__NSDictionaryM: 0x30155d740> was mutated while being enumerated.

### ISSUE THREAD

**Issue by bsebire-hw** [Instabug/Instabug-iOS#500]
Hello,

We are trying Instabug on our production application for a few days and we notice a lot of users encountered a crash during the app startup which seems related to Instabug.

Here is the stacktrace : 
```
NSGenericException : *** Collection <__NSDictionaryM: 0x30155d740> was mutated while being enumerated.
-[IBGFeatures notifyDelegatesWithResponse:] + 171
```

```
0    CoreFoundation                           0x194c68f20     __exceptionPreprocess + 164
1    libobjc.A.dylib                          0x18caee018     objc_exception_throw + 59
2    CoreFoundation                           0x194d6720c     __NSFastEnumerationMutationHandler + 115
3    Instabug                                 0x102c17758     -[IBGFeatures notifyDelegatesWithResponse:] + 171
4    Instabug                                 0x102c1766c     -[IBGFeatures updateFeaturesWithFeaturesResponse:] + 4431
5    Instabug                                 0x102a69b0c     -[IBGFeaturesNetworkManager featuresRequestDone:isSuccess:error:response:shouldUpdateFeatures:] + 271
6    Instabug                                 0x102a969d8     -[IBGFeaturesRequest networkDone:URLSessionDataTask:response:error:responseDictionary:] + 395
7    Instabug                                 0x102b8afa0     __51-[IBGNetwork URLSession:task:didCompleteWithError:]_block_invoke_2 + 199
8    libdispatch.dylib                        0x19cb0c13c     _dispatch_call_block_and_release + 31
9    libdispatch.dylib                        0x19cb0ddd4     _dispatch_client_callout + 19
10   libdispatch.dylib                        0x19cb1c5a4     _dispatch_main_queue_drain + 987
11   libdispatch.dylib                        0x19cb1c1b8     _dispatch_main_queue_callback_4CF + 43
12   CoreFoundation                           0x194c3b710     __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 15
13   CoreFoundation                           0x194c38914     __CFRunLoopRun + 1995
14   CoreFoundation                           0x194c37cd8     CFRunLoopRunSpecific + 607
15   GraphicsServices                         0x1d9ae81a8     GSEventRunModal + 163
16   UIKitCore                                0x19727090c     -[UIApplication _run] + 887
17   UIKitCore                                0x1973249d0     UIApplicationMain + 339
18   HelloWork                                0x100eec068     0x100ee4000
```

### Steps to Reproduce the Problem
We didn't reproduce the problem.

### Expected Behavior
Instabug doesn't make the app crash.

### Actual Behavior
Instabug seems to crash during startup.

### SDK Version
13.2.0

### iOS Version
At least : 
- 15.1.1
- 15.3.1
- 16.2
- 17.4.1
- 17.5.1
- 17.6
- 18.0

### Device Model
At least : 
- iPhone 15 Pro
- iPhone 15 Pro Max
- iPhone 13
- iPhone 14 Pro Max
- iPhone 12
- iPhone 15
- iPhone 14
- iPhone 14 Pro
- iPhone 11
- iPhone 13 Pro
- iPhone 12 Pro
- iPhone 13 Pro Max
- iPhone 14 Plus
- iPhone 15 Plus
- iPad Air 2 (Wi-Fi)
- iPad air 4th Gen (WiFi)
- iPhone 12 Mini
- iPhone 13 Mini

Thanks for your help !

**Comment by agustindeleon7** [comment]
+1

**Comment by AhmedAshraf605** [comment]
@bsebire-hw @agustindeleon7  We are working on a solution and will release it soon.

**Comment by MichaelJoo** [comment]
I am facing the same issue, would appreciate if this would be resolved soon? I am currently using free version.

**Comment by AhmedAshraf605** [comment]
HI @bsebire-hw @agustindeleon7 @xmollv @MichaelJoo 
This should be fixed in 13.3.0

**Comment by MichaelJoo** [comment]
I have already updated to 13.3.0 SDK a few days ago - thanks so much for the help.
